src/templates/pt/2019/contributors.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base/2019/contributors.html" %}
+
+{% block title %}Colaboradores de 2019 | O Web Almanac por HTTP Archive{% endblock %}
+
+{% block description %}O {{ config.contributors.items() | length }} pessoas que contribuíram para o Web Almanac 2019 como analistas, autores, pensadores, designers, desenvolvedores, editores, revisores e tradutores.{% endblock %}
+
+{% block filter_by_team %}Filtrar por equipe: <span id="filtered-contributors">{{ self.contributors() }}</span><span id="contributors-total-text" class="hidden"> de <span id="contributors-total">{{ config.contributors.items() | length }}</span></span> contribuidores.{% endblock %}
```

Comment:
Added these in #1166 

```suggestion
{% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
{% block date_modified %}2020-07-17T00:00:00.000Z{% endblock %}

{% block filter_by_team %}Filtrar por equipe: <span id="filtered-contributors">{{ self.contributors() }}</span><span id="contributors-total-text" class="hidden"> de <span id="contributors-total">{{ config.contributors.items() | length }}</span></span> contribuidores.{% endblock %}
```

Reply:
I thought they could be removed, because they had conflict